fw/cache.c

```diff
@@ -270,13 +272,20 @@ static TfwStr g_crlf = { .data = S_CRLF, .len = SLEN(S_CRLF) };
 		u_end = (uri_path)->chunks + (uri_path)->nchunks;	\
 	}								\
 	if (!TFW_STR_EMPTY(host)) {					\
```

Comment:
`if (!TFW_STR_EMPTY(host))` must always be true, otherwise we have wrong logic. We even can replace it with `BUG_ON(TFW_STR_EMPTY(host));` because key always must consist of two parts uri and host.